RELEASE-NOTES.md

```diff
@@ -9,6 +9,7 @@ This is a bugfix release.
 * Add `log.access` switch to disable access logging ([#1109](https://github.com/basho/riak_cs/issues/1109)/[PR#1115](https://github.com/basho/riak_cs/pull/1115)).
 * Add missing riak-cs.conf items:` max_buckets_per_user` and `gc.batch_size` ([#1109](https://github.com/basho/riak_cs/issues/1109)/[PR#1115](https://github.com/basho/riak_cs/pull/1115)).
 * Fix bugs around subsequent space characters for Delete Multiple Objects API and user administration API with XML content ([#1129](https://github.com/basho/riak_cs/issues/1129)/[PR#1135](https://github.com/basho/riak_cs/pull/1135)).
+* Fix `URL path` resource and query parameters to work in AWS v4. Previously, '+' was being input instead of '%' for blank spaces. ([PR#1141](https://github.com/basho/riak_cs/pull/1141))
```

Comment:
I'd rather recommend markdown notation, like  "Previously, `+` was being input instead of `%20` for blank spaces."